app/addons/replication/__tests__/helpers.tests.js

```diff
@@ -15,11 +15,11 @@ import helpers from '../helpers';
 describe('Replication Helpers - getDatabaseLabel', () => {
 
   it('returns database name for string', () => {
-    const db = 'http://tester:testerpass@127.0.0.1/fancy/db/name';
+    const db = 'http://tester:testerpass@127.0.0.1/db/fancy%2Fdb%2Fname';
 
     const res = helpers.getDatabaseLabel(db);
 
-    expect(res).toBe('fancy/db/name');
+    expect(res).toBe('fancy%2Fdb%2Fname');
```

Comment:
We have to pass the database name encoded otherwise it's hard to extract the database name with relative urls.